packages/apps/package.json

```diff
@@ -29,7 +29,7 @@
     "ethlint": "1.2.4",
     "mocha": "6.1.4",
     "solc": "0.5.8",
-    "truffle": "5.0.10",
+    "truffle": "5.0.19",
```

Comment:
?

Reply:
I had an issue that I thought was truffle related so I tried bumping the version. Turned out it wasn't truffle-related. I'll roll this back.